scripts/docstrings_checker.py

```diff
@@ -236,7 +236,7 @@ def get_args_list_from_function_definition(cls, function_node):
         """
         # Ignore self and cls args.
         args_to_ignore = ['self', 'cls']
-        return python_utils.get_args_of_function_node(
+        return util.get_args_of_function_node(
```

Comment:
Please can you verify if `ASTDocStringChecker` is used anywhere? I think we previously discussed that it is not needed.

Reply:
Not used so removed entire class